contributors/design-proposals/storage/volume-topology-scheduling.md

```diff
@@ -383,19 +732,19 @@ PrioritizeUnboundPVCs(pod *v1.Pod, filteredNodes HostPriorityList) (rankedNodes
 
 TODO (beta): figure out weights and exact calculation
 
-##### Assume
+#### Assume
 Once all the predicates and priorities have run, then the scheduler picks a
 Node.  Then we can bind or provision PVCs for that Node.  For better scheduler
 performance, we’ll assume that the binding will likely succeed, and update the
-PV cache first.  Then the actual binding API update will be made
+PV and PVC caches first.  Then the actual binding API update will be made
 asynchronously, and the scheduler can continue processing other Pods.
 
-For the alpha phase, the AssumePVCs function will be directly called by the
+For the alpha phase, the AssumeVolumes function will be directly called by the
 scheduler.  We’ll consider creating a generic scheduler interface in a
 subsequent phase.
 
 ```
-AssumePVCs(pod *v1.Pod, node *v1.Node) (pvcBindingRequired bool, err error)
+AssumeVolumes(pod *v1.pod, node *v1.node) (pvcbindingrequired bool, err error)
 ```
 1. If all the Pod’s PVCs are bound, return false.
```

Comment:
I think the rough logic of AssumePodVolumes would be : 

- Could returns true if all volumes are fully bound
- Could returns false if any volume binding operation needs to be done afterward, and then the pod will be enqueued into _BindQueue_